audits/org-audit-snapshots/2026-08-29-preserved-wip-reconciliation.md

```
@@ -0,0 +1,87 @@
+
+The inventory was produced without ref mutation using:
+
+```zsh
+git for-each-ref refs/remotes/origin --no-merged=origin/main
+git log origin/main --format='%H%x09%s'
+gh api --paginate 'repos/KooshaPari/ResearchLedger/pulls?state=all&per_page=100' \
+  --jq '.[] | [.number, .state, .merged_at, .head.sha] | @tsv'
+```
+
+The paginated PR query returned 70 historical records at collection time. PR
+#72 is intentionally excluded from that count because it was created to review
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)